src/app/learn/get-started/advanced/sticky-settings/index.html

```diff
@@ -5,98 +5,13 @@
 
   <stache-page-summary>
     <p>
-      Sticky settings enable you to save user settings across sessions in grids and tile dashboards. Sticky settings preserve column order and the sorting of columns in grids, and they preserve the layout and collapsed state of tiles in tile dashboards.
+      Sticky settings enable you to save user settings across sessions in flyouts, grids, and tile dashboards.
```

Comment:
Maybe add "and custom components/services," or something like that?

Reply:
I added ", as well as in custom components and services" to the end of the sentence and will push the changes up shortly.